platform/darwin/src/MGLStyleLayer.h.ejs

```diff
@@ -0,0 +1,29 @@
+<%
+  const type = locals.type;
+  const layoutProperties = locals.layoutProperties;
+  const paintProperties = locals.paintProperties;
+-%>
+// This file is generated. Do not edit.
```

Comment:
We should strip out this comment when packaging the SDK. Also say what generates the file, or how to generate it.